config/locales/overrides/decidim.en.yml

```diff
@@ -3,3 +3,11 @@ en:
     menu:
       processes: 'Participate'
       more_information: 'About'
+    proposals:
+      equity:
+        card_tag_title: 'Equity score:'
```

Comment:
I wonder if we should instead call this "Equity impact"? Having a score makes it non-obvious if a high or low score is good (as we saw with some confusion with the sorting).

Reply:
+1 for "equity impact". Also since it's going to be displayed like "Equity impact: Highest" should that second word be title-cased? (i.e., "Equity Impact: Highest")

Reply:
I changed the copy here based on feedback.

My opinion is that more thought will need to be put into how to contextualize this information. A dog park in a poor neighborhood will be generally used by privileged folks and has a low equity impact compared to something like a pedestrian safety improvement. So I assume the language will need to be changed after this